GoogleChromeLabs/ps-analysis-tool
# Remove topics list feature

## Description

Temporarily revert topics list added in https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/159 until we have a solution for the attestation error

![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/6297436/182bf129-a57b-4d39-862f-b810a620cf58)


## Relevant Technical Choices

- Remove all code and data related to topics list

## Testing Instructions

- Go to "Private Advertising" > Topics section in the PSAT extension devtool and you should not see the topics list.

## Additional Information:



## Screenshot/Screencast

<img width="1440" alt="image" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/6297436/f4f551b6-6d37-480d-92b9-18aec8409f49">


---

## Checklist






Fixes #
